samples/bluetooth/mesh_demo/src/microbit.c

```diff
@@ -129,13 +129,14 @@ void board_play_tune(const char *str)
 		}
 
 		if (period) {
-			pwm_set_usec(pwm, BUZZER_PWM_CHANNEL, period, period / 2U, 0);
+			pwm_set(pwm, BUZZER_PWM_CHANNEL, PWM_USEC(period),
+				PWM_USEC(period / 2U), 0);
```

Comment:
```suggestion
				PWM_USEC(period) / 2U, 0);
```
to get e.g. 1702500 instead of 1702000 when `period` is 3405. No one will actually hear the difference in these samples, but in case their code was to be taken as a pattern, it would be better to avoid integer truncation.

Reply:
adjusted, thanks